src/main/resources/application.yml

```diff
@@ -1,13 +1,13 @@
 spring:
   datasource:
-    url: jdbc:mysql://localhost/orlp?useSSL=false
+    url: jdbc:mysql://localhost/orlp?createDatabaseIfNotExist=true&useSSL=false
```

Comment:
`createDatabaseIfNotExist=true` do we need to change it?

Reply:
It seems, that yes, because when we start app server, it tries to connect to database ORLP. 
And, if database doesn't exist, it will throw MySQLException.